addons/account/tests/common.py

```diff
@@ -208,6 +208,9 @@ def create_accounting_minimal_data(cls):
             'user_type_id': cls.env.ref('account.data_account_type_equity').id,
         })
 
+        cls.company.income_currency_exchange_account_id = cls.income_fx_income.id
+        cls.company.expense_currency_exchange_account_id = cls.income_fx_expense.id
```

Comment:
This setup is about to be removed today

Reply:
Should I wait for it to be removed and then fix the tests ?